src/harness/testcases/testdata/pal_db/pal_db_record.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This should be a list of PAL DB records (i.e. there should be `[]` on the outermost level). Please ensure that the file generated by the default config follows that format, but please do not commit that file.

Reply:
We have doubled checked it. This statement 
json.dumps(pal_records_a,pal_records_b,indent=2)
write a list of PAL records.